### PR TITLE
Enhancement: Clean up .gitignore

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] cleans up `.gitignore`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-composer.lock
-composer.phar
-phpunit.xml
-phpmd.xml
-phpdox.xml
-clover.xml
-example/test-data
-phing-latest.phar
-build/
+/build/
```
Comment (localheinz) [inline .gitignore]: We probably only want to ignore files in the root directory.

Appended `/` to make it more obvious which is expected to be a directory and which a file.

Sorted by

- kind (directory or file)
- name

File: .gitignore [hunk 2]
```diff
@@ -1,10 +1,9 @@
-vendor
-composer.lock
-composer.phar
-phpunit.xml
-phpmd.xml
-phpdox.xml
-clover.xml
-example/test-data
```
Comment (localheinz) [inline .gitignore]: Does not exist.

**Comment by Ocramius** [comment]
👍

**Comment by localheinz** [comment]
Thank you, @Ocramius!

**Pull request merged**